### DISCUSSION TITLE
Installation error "libgfortran.so.4 not found" on Ubuntu 20.04

### DISCUSSION THREAD

**Discussion by ngrilli** [neperfepx/neper#298] [Q&A] [answered]
When installing version 4.2.0 on Ubuntu 20.04 using:

cmake ..
make
sudo make install
make test

All the tests were failing.
When executing neper, for instance:

neper --version

I was getting the error:

libgfortran.so.4 not found 

I am answering myself,
hope this is useful to others.

Nicolò Grilli

**Comment by ngrilli** [comment, marked as answer]
I did:

sudo apt install gfortran-7

and do the installation again,
now Neper 4.2.0 works fine

Nicolò Grilli

**Reply by rquey** [reply]
As a side note, you (probably) should not need gfortran to install Neper (which is C/C++)... something we should fix.
